src/main/java/racing/CarRacing.java

```diff
@@ -12,21 +14,28 @@ public static void main(String[] args) {
     }
 
     private void start() {
-        prepare();
-        int rounds = InputView.inputRounds();
-        play(rounds);
+        participateCars();
+        play();
     }
 
-    private void play(int rounds) {
+    private void play() {
+        int rounds = InputView.inputRounds();
         ResultView.printResultTitle();
         for (int i = 1; i <= rounds; i++) {
             playRound();
-            announceResult(i);
+            announceRoundResult(i);
         }
+        announceWinner();
     }
 
-    private void prepare() {
-        participateCars();
+    private void announceWinner() {
+        List<String> winnerNames = findWinners();
+        ResultView.printWinners(winnerNames);
+    }
+
+    private List<String> findWinners() {
+        int maxDistance = cars.stream().mapToInt(Car::getDistance).max().getAsInt();
+        return cars.stream().filter(car -> car.getDistance() == maxDistance).map(car -> car.getName()).collect(Collectors.toList());
```

Comment:
```suggestion
        int maxDistance = cars.stream()
            .mapToInt(Car::getDistance)
            .max()
            .getAsInt();
        return cars.stream()
            .filter(car -> car.getDistance() == maxDistance)
            .map(car -> car.getName())
            .collect(Collectors.toList());
```
stream이 한 줄로 되어 있으면 생각보다 읽기 쉽지 않더라구요. 개인적으로는 여러 줄에 나눠쓰는게 읽기 편하다고 생각하는데 민지님의 생각은 어떠신가요?

Reply:
동의합니다.